solutions/kubernetes/ceph-binary-build.sh

```diff
@@ -144,7 +144,7 @@ function ceph_build() {
                 popd
 
                 add_common_separator "List generated binary packages (*.deb)"
-                ls *.deb
+                ls ./*.deb
```

Comment:
Please check that scripts works with this change

Reply:
Removed this file